README.md

```diff
@@ -10,15 +10,6 @@ QCEngine
 
 Quantum chemistry program executor and IO standardizer ([QCSchema](https://github.com/MolSSI/QCSchema)) for quantum chemistry.
 
-Available Compute Programs:
```

Comment:
/why remove this block from the README? Should it not keep expanding as we get more?

Reply:
Best to keep it in the primary docs (IMO) so that we do not have two copies running around. Happy to change this as desired.

Reply:
Good reason. Keep it as is then